### PR TITLE
ED-2553 input validation contact us

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: contact/forms.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from django import forms
+from directory_validators.company import keywords_special_characters
```
Comment (konradko) [inline contact/forms.py]: we should rename the validator to something like `special_characters` - `keywords` does not fit in this context

Comment (konradko) [reply contact/forms.py]: also `'You can only enter letters, numbers and commas.'` - we can expect more punctuation marks in the feedback. So probably need a new validator that would prevent html/js + emails and websites (that one we already have for message-to-supplier)

Comment (sdonk) [reply contact/forms.py]: Do we have that validator somewhere already?

I'll add a ticket to change the validators name in company to something more generic

**Comment by sdonk** [comment]
@konradko can you please review my changes?

File: help/urls.py [hunk 2]
```diff
@@ -1,8 +1,5 @@
 from django.conf.urls import url, include
 from django.contrib import admin
-from django.conf import settings
-
```
[the remark this answers is not in the document]
Comment (konradko) [reply help/urls.py]: we should add the static analysis we have on directory repos here later

**Pull request merged**